src/components/BorrowerProfile/LendersAndTeams.vue

```diff
@@ -239,6 +239,7 @@ export default {
 			observer: null,
 			itemQueryLimit: 20,
 			itemQueryOffset: 0,
+			sortBy: 'overallLoanedAmount',
```

Comment:
This should also be declared and setup in the Teams query (line 165) and used on line 172.

Then you can conditionally apply `this.storyBy` in the extracted variables block down in the fetchItems method.

Reply:
@mcstover Aren't these changes already present in this PR? The Teams query is on line 145, where I've added the sortBy option.